### PR TITLE
Handle null values gracefully in DropwizardExports

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

File: simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
```
Comment (brian-brazil) [inline simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java]: This isn't used.

Comment (henri-tremblay) [reply simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java]: Right

**Comment by henri-tremblay** [comment]
Unit test added

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**